src/Cohesity.Powershell/Scripts/Recover-CohesityBackupToView.ps1

```diff
@@ -45,13 +42,16 @@ function Recover-CohesityBackupToView {
         $protectionSourceId = $null
         $sourceName = $null
         $startedTimeUsecs = $null
+        $errorMeaaage = $null
         foreach ($item in $resp.objectSnapshotInfo) {
             # $json = $item | ConvertTo-Json
             # Write-Host $json
-            if($item.environment -ne "kGenericNas" -AND $item.environment -AND "kNetapp" -AND $item.environment -ne "kIsilon") {
+            if($item.snapshottedSource.environment -ne "kGenericNas" -AND $item.snapshottedSource.environment -ne "kNetapp" -AND $item.snapshottedSource.environment -ne "kIsilon") {
```

Comment:
Can you spend sometime and see if we can include isilon and netapp envs too ?

Reply:
Done. Tested on netapp and isilon (10.2.37.80) sources

Reply:
Done. Tested on netapp and isilon